public/pages/CreateMonitor/containers/CreateMonitor/utils/formikToMonitor.js

```diff
@@ -30,14 +32,7 @@ export function formikToMonitor(values) {
     type: 'monitor',
     enabled: !values.disabled,
     schedule,
-    inputs: [
-      {
-        search: {
-          indices,
-          query,
-        },
-      },
-    ],
+    inputs: [isHttp ? http : search],
```

Comment:
If isHttp there seems to be no need to call `formikToSearch` above. Might make more sense to just have a `formikToInputs` where you then return the inputs array. Inside `formikToInputs` you can check the http status and return the http type or search type.

Reply:
Makes sense, changed to `formikToInputs()`.